packages/docs/content/docs/developer-updates/2025-12-19-weekly-update-15.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Weekly Update #15
+---
+
+# Weekly Update #15
+
+### TL;DR
+
+This week includes several improvements and bug fixes across the VSCode extension and Web UI. 
+
+A mix of workflow enhancements, UI polish, and reliability fixes landed. Here are the highlights:
+
+
+### ✨ Enhancements
+
+- **Improved Worktree Selector in VSCode Web UI:** Added support for creating a new worktree directly from the dropdown, with a clearer “New Worktree” option and updated icon. Also includes tooltips and improved labeling for a smoother workflow. **[#896](https://github.com/TabbyML/pochi/issues/896)**
```

Comment:
We already have dropdown with creating a new worktree. The feature is more about creating worktree with generated name.